requirements.txt

```diff
@@ -5,3 +5,4 @@ docopt
 requests_cache
 pytablewriter
 publicsuffix
+pyopenssl=17.2.0
```

Comment:
This was my addition - you noted a bug with this in #87, which I'll fix (and add a check for to our CI build process).